galaxy_ng/app/tasks/publishing.py

```diff
@@ -61,6 +67,14 @@ def import_and_move_to_staging(temp_file_pk, **kwargs):
         call_copy_task(collection_version, inbound_repo, staging_repo)
         call_remove_task(collection_version, inbound_repo)
 
+        if settings.GALAXY_ENABLE_API_ACCESS_LOG:
+            api_access_log.info(
+                "Anonymous uploaded collection: %s-%s-%s",
```

Comment:
Since only authenticated users can import a collection, is there a reason we are saying "Anonymous..." versus "Collection uploaded: ..."?

Reply:
I don't believe we have access to the `request` object to determine who the user is, hence the `Anonymous`

Reply:
I think this should be `"Collection uploaded: %s-%s-%s"` - folks reading the log may think the app allows anonymous users to access the system and uploaded a collection.